SpatialGDK/Source/Private/Interop/SpatialSender.cpp

```diff
@@ -233,15 +257,21 @@ void USpatialSender::SendRotationUpdate(Worker_EntityId EntityId, const FRotator
 	Connection->SendComponentUpdate(EntityId, &Update);
 }
 
-void USpatialSender::SendRPC(UObject* TargetObject, UFunction* Function, void* Parameters, bool bOwnParameters)
+void USpatialSender::SendRPC(TSharedRef<FPendingRPCParams> Params)
 {
-	FClassInfo* Info = TypebindingManager->FindClassInfoByClass(TargetObject->GetClass());
+	if (!Params->TargetObject.IsValid())
+	{
+		// Target object was destroyed before the RPC could be (re)sent
+		return;
+	}
+
+	FClassInfo* Info = TypebindingManager->FindClassInfoByClass(Params->TargetObject->GetClass());
 	if (Info == nullptr)
 	{
 		return;
 	}
 
-	FRPCInfo* RPCInfo = Info->RPCInfoMap.Find(Function);
+	FRPCInfo* RPCInfo = Info->RPCInfoMap.Find(Params->Function);
 	check(RPCInfo);
 
 	Worker_EntityId EntityId = 0;
```

Comment:
Nit which is not related to this PR. Could this be changed to `Worker_EntityId EntityId = SpatialConstants::INVALID_ENTITY_ID;`?